lib/travis/build/appliances/fix_sudo_enabled_trusty.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'travis/build/appliances/base'
+
+module Travis
+  module Build
+    module Appliances
+      class FixSudoEnabledTrusty < Base
+        def apply
+          sh.if sh_is_linux? do
+            sh.if sh_is_trusty? do
+              sh.cmd 'unset _JAVA_OPTIONS', echo: false
+              sh.cmd 'unset MALLOC_ARENA_MAX', echo: false
+            end
+          end
+        end
+
+        def apply?
+          !data.disable_sudo?
+        end
+
+        private
+
+        def sh_is_linux?
```

Comment:
What is the purpose of `sh_` prefix?

Reply:
Mmmmm I vaguely recall meaning to indicate that the method is generating some shell goop as opposed to answering in ruby boolean.  This bit is copy pasta from the other appliance for fixing container-based trusty.

Reply:
ah, I see. I agree that it would be a good idea to standardize some function naming convention. I suppose we can start from here.